samples/python/django/sample.py

```diff
@@ -6,86 +6,117 @@
 
+
 def create_sample_album(album_id, singer=None):
   global x
   x += 1
   return Album(id=album_id,
                singer=singer,
-               title='album'+str(x),
-               marketing_budget = 200000,
-               release_date = datetime.date.today(),
-               cover_picture= b'hello world',
+               title='album' + str(x),
+               marketing_budget=200000,
+               release_date=datetime.date.today(),
+               cover_picture=b'hello world',
                created_at=datetime.datetime.now(pytz.UTC),
                updated_at=datetime.datetime.now(pytz.UTC))
 
-def create_sample_track(track_id, track_number, album = None):
+
+def create_sample_track(track_id, track_number, album=None):
   global x
   x += 1
   return Track(track_id=track_id,
                album=album,
                track_number=track_number,
-               title='track'+str(x),
+               title='track' + str(x),
                sample_rate=124.543,
                created_at=datetime.datetime.now(pytz.UTC),
                updated_at=datetime.datetime.now(pytz.UTC))
 
+
 def create_sample_venue(venue_id):
   global x
   x += 1
   return Venue(id=venue_id,
-               name='venue'+str(x),
-               description='description'+str(x),
+               name='venue' + str(x),
+               description='description' + str(x),
                created_at=datetime.datetime.now(pytz.UTC),
                updated_at=datetime.datetime.now(pytz.UTC))
 
-def create_sample_concert(concert_id, venue = None, singer = None):
+
+def create_sample_concert(concert_id, venue=None, singer=None):
   global x
   x += 1
   return Concert(id=concert_id,
                  venue=venue,
                  singer=singer,
-                 name='concert'+str(x),
+                 name='concert' + str(x),
                  start_time=datetime.datetime.now(pytz.UTC),
-                 end_time=datetime.datetime.now(pytz.UTC)+datetime.timedelta(hours=1),
+                 end_time=datetime.datetime.now(pytz.UTC) + datetime.timedelta(
+                   hours=1),
                  created_at=datetime.datetime.now(pytz.UTC),
                  updated_at=datetime.datetime.now(pytz.UTC))
 
+
 def create_tables():
   file = open('create_data_model.sql', 'r')
   ddl_statements = file.read()
-  print(ddl_statements)
+  # print(ddl_statements)
   with connection.cursor() as cursor:
     cursor.execute(ddl_statements)
 
+
 @atomic(savepoint=False)
 def add_data():
-  singer = create_sample_singer('1')
-  singer.save()
-
-  album = create_sample_album('1', singer)
-  album.save()
-
-  track = create_sample_track('1', '2', album)
-  track.save(force_insert=True)
-
-  venue = create_sample_venue('1')
-  venue.save()
+  try:
+    singer = create_sample_singer('1')
+    singer.save()
+
+    singer_object = Singer.objects.get(id=singer.id)
+    if singer_object.first_name != singer.first_name or singer_object.last_name != singer.last_name:
```

Comment:
Here and elsewhere: As this is a sample, I think it would be better to just print the singer properties to standard out, and then check for that output in the test case. The sample should as much as possible be aimed at showing how a user can use Django with PGAdapter, and contain as little test code as possible.